bind.go

```diff
@@ -153,10 +153,14 @@ func bind(server func(jid.JID, string) (jid.JID, error)) StreamFeature {
 				if err != nil && !ok {
 					return mask, nil, err
 				}
+				bindIqNamespace := stanza.NSClient
```

Comment:
Another nit: in Go convention is to use the same case for acronyms, so this should be `bindIQNamespace`

Reply:
I wasn't aware of this convention, thanks!
I've amended the commit with the changes you requested

Reply:
Thanks!